tests/conftest.py

```
@@ -39,14 +39,15 @@ def pytest_configure(config: pytest.Config) -> None:
-        "model_outputs=None, expected_stdout=None, config_content=None, **kwargs): "
+        "model_outputs=None, expected_stdout=None, config_content=None, aliases=None, **kwargs): "
         "Mark test as CLI documentation source. "
-        "Either golden_output, version_outputs, model_outputs, or expected_stdout is required.",
+        "Either golden_output, version_outputs, model_outputs, or expected_stdout is required. "
+        "aliases: list of alternative option names (e.g., ['--capitalise-enum-members']).",
     )
     config._cli_doc_items: list[dict[str, Any]] = []
 
 
-def _validate_cli_doc_marker(node_id: str, kwargs: dict[str, Any]) -> list[str]:  # noqa: ARG001, PLR0912  # pragma: no cover
+def _validate_cli_doc_marker(node_id: str, kwargs: dict[str, Any]) -> list[str]:  # noqa: ARG001, PLR0912, PLR0914  # pragma: no cover
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLR0912`, `PLR0914`)

Remove unused `noqa` directive

(RUF100)